Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -95,10 +92,12 @@ void Start()
                 }
                 else
                 {
-                    themeColorController.SetProgramVariable("audioLink", audioLink);
+                    themeColorController.audioLink = audioLink;
                     themeColorController.SendCustomEvent("UpdateAudioLinkThemeColors");
```

Comment:
Does this still work now that the field has type `ThemeColorController`? This is basically equivalent to a method call, right?

Reply:
[ffa9c3a](https://github.com/llealloo/vrc-udon-audio-link/pull/203/commits/ffa9c3ae9054efa9f68658f2392f442820b4267e)